protocol-tokio/src/protocol/codec/message.rs

```diff
@@ -171,6 +178,11 @@ impl Message {
             MessageType::MsgBlock => {
                 Ok(Message::Block(lwcid, cbor.deserialize_complete().unwrap()))
             }
+            MessageType::MsgSubscribe1 => {
+                let v : u64 = cbor.deserialize_complete().unwrap();
+                let keep_alive = v == 43;
```

Comment:
This silently treats all values other than 43 or 42 as "non-keepalive". I hope it's OK.